README.md

```diff
@@ -116,16 +126,26 @@ vary: origin,accept-encoding
 }
 ```
 
-The `Accept-Encoding:` header included in the request unsets the default header `Accept-Encoding: gzip, deflate`.
+The `Accept-Encoding:` header included in the request unsets `httpie`'s default
+header `Accept-Encoding: gzip, deflate`.
 
-Any contract violations will be returned in a `s1-violations` header, which will contain a JSON object with all violations found in the response. You can also use the `--errors` flag which will turn any request or response violation found into a [RFC7807](https://tools.ietf.org/html/rfc7807) machine readable error.
+Any contract violations will be returned in a `s1-violations` header, which will
+contain a JSON object with all violations found in the response. You can also
+use the `--errors` flag which will turn any request or response violation found
+into a [RFC7807](https://tools.ietf.org/html/rfc7807) machine readable error.
 
 ## Compatibility with Community Tooling
 
-A lot of tooling for working with OpenAPI specs does not support the full specification. In particular, many tools do not support multiple files specs. To use such a tool, bundle the spec into a single file using `make bundle`.
+A lot of tooling for working with OpenAPI specs does not support the full
+specification. In particular, many tools do not support multiple files specs. To
+use such a tool, bundle the spec into a single file using `make bundle`.
 
-The tool used by `make bundle` can do much more than bundle a multiple file spec into a single file. It can convert specs between `YAML` and `JSON`, fully dereference a spec, and more.
+The tool used by `make bundle` can do much more than bundle a multiple file spec
+into a single file. It can convert specs between `YAML` and `JSON`, fully
+dereference a spec, and more.
 
 ## See Also
 
 The API tooling Notion page.
```

Comment:
Not sure if you wanted to add a link to the Open API tools notions page (https://www.notion.so/lob/OpenAPI-tooling-e17f5a864a4a48d1886dcd95b53cf694)